### PR TITLE
[Spanish] translate server-side-rendering.md

### PR DESCRIPTION
**Pull request by josewhitetower**

### CODE REVIEW

**Comment by debs-obrien** [comment]
This has also been translated here https://github.com/nuxt/nuxtjs.org/pull/751 so perhaps checking this also against the other one and we can merge either of them depending on if need further review etc. cc @alvarosaburido @Dawntraoz

File: content/es/guides/concepts/server-side-rendering.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+```
+
+## Pasos para la renderización del lado del servidor con Nuxt.js
+
+### Paso 1: Del navegador al servidor
+
+Cuando un navegador envía la solicitud inicial, esta alcanzará al servidor interno de Node.js. Nuxt.js generará el HTML y lo enviará de vuelta al navegador con los resultados de las funciones ejecutadas, por ejemplo, `asyncData`, `nuxtServerInit` o `fetch`. Las funciones de hook también se ejecutan.
+### Step 2: Server to Browser
+
+The browser receives the rendered page El navegador recibe la página renderizada del servidor con el HTML generado. El contenido se muestra y la hidratación del Vue.js se activa, haciéndolo reactivo. Después de este proceso, la página es interactiva.d HTML.
```
Comment (Kronhyx) [inline content/es/guides/concepts/server-side-rendering.md:94]:
```suggestion
El navegador recibe la página renderizada del servidor con el HTML generado. El contenido se muestra y la hidratación del Vue.js se activa, haciéndolo reactivo. Después de este proceso, la página es interactiva.d HTML.
```

**Review by Kronhyx — CHANGES_REQUESTED** [review]

**Comment by debs-obrien** [comment]
Hi, great work @josewhitetower . As this had also been translated in another pr which is now merged have a look at the conflicts and see if it now makes sense to fix any of the conflicts and merge them or close the pr. Thanks

**Comment by josewhitetower** [comment]
The merged translation seems great to me, I will close this pr :)

**Pull request closed without merging**